src/org/exist/xquery/value/Type.java

```diff
@@ -189,7 +189,7 @@
         //TODO use parentheses after the nodes name  ?
         defineBuiltInType(NODE, "node()");
         defineBuiltInType(ITEM, "item()");
-        defineBuiltInType(EMPTY, "empty()");
+        defineBuiltInType(EMPTY, "empty-sequence()","empty");
```

Comment:
it must be `empty()`

Reply:
My mobile app did not su the correct comment' you are right, I'll update.
Te question remains... is this the correct change? is it an external interface change which means we need to hold it till eXist-db 4.0.0 ?

Reply:
no needs to wait till 4.x as it support both, remove `empty()` or `map` will require 4.x